### ISSUE TITLE
create a components/re-usables system for text in the docs

### ISSUE THREAD

**Issue by HonkingGoose** [renovatebot/renovate#8341]
**What would you like Renovate to be able to do?**

I notice that we're repeating ourselves sometimes in the documentation.

We have really similar explanations for how `automergeType=branch` works in these 3 files:
- `automerge-configuration.md`
- `configuration-options.md`
- `dependency-pinning.md`

It would be good to have just one explanation, which we can then "reuse" multiple times where needed.

I think there are probably other places where we are duplicating our text, but I cannot quantify this.

**Did you already have any implementation ideas?**

Basically we should find a way to make a "component" out of really common pieces of text. We then plugin the component each time we need it.

I think Docusaurus v2 supports using components, as it uses React behind the scenes, and React has components.

See the React docs on components: https://reactjs.org/docs/react-component.html

So this could be something that we can look at after we've migrated to Docusaurus v2. 😄

**Comment by viceice** [comment]
Time to go futher with #7578 🙃

**Comment by HonkingGoose** [comment]
I think we're `status:blocked` on this one until we get Docusaurus v2 running?

**Comment by HonkingGoose** [comment]
Closing this issue, as it's gone stale. We decided to keep using MkDocs, which doesn't support components.

We should still work out a good structure for our docs, where we don't need to duplicate information. We should instead link to the canonical source for each topic, where it makes sense to do so.
